src/lib/dnssd/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -160,22 +160,19 @@ class AdvertiserMinMdns : public ServiceAdvertiser,
         Optional<uint32_t> mrpRetryIntervalIdle, mrpRetryIntervalActive;
         params.GetMRPRetryIntervals(mrpRetryIntervalIdle, mrpRetryIntervalActive);
         // TODO: Issue #5833 - MRP retry intervals should be updated on the poll period value change or device type change.
-#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
-        if (chip::DeviceLayer::ConnectivityMgr().GetThreadDeviceType() ==
```

Comment:
Shouldn't we keep the runtime check? I think if there's API for disabling SED at runtime we shouldn't assume that SED is always active if CHIP_DEVICE_CONFIG_ENABLE_SED

Reply:
As discussed F2F it might not be the real case. We will do the follow-up if such appears.